CONTRIBUTING.md

```diff
@@ -127,6 +127,25 @@ reference GitHub issues that this commit **Closes**. In order to maintain a refe
 `fixes #<issue_number>` if it closes a related issue or `issue #<issue_number>`
 if it's a partial fix.
 
+## What should you do once your PR is merged?
+
+Once your PR is merged, it's recommended to pull new changes from the `main` (which will also include the one that you made).
+So, for that purpose specify a new remote `upstream` repository which will be pointing to the actual [mozilla/fxa](https://github.com/mozilla/fxa) repo.
```

Comment:
I'm not sure if I see a note anywhere about contributors pushing to and PR'ing from their fork. FxA team members (part of the Mozilla organization) can push upstream, but contributors must push to their fork and put a PR in from their fork. It might be worth explicitly calling that out in another section? 🙂

Also, I think this can maybe be very slightly rephrased (around "new remote `upstream` repository"), since I'd still consider it the same repository, just a different remote. Maybe just lose the word "repository" so it reads "...specify a new remote `upstream` which will..."?

Reply:
Ohhh my bad that it came up like that. 
I created this PR just with the intention of setting a new remote to fetch new updates from. I could close it right away if it's not needed?
Yeah, it completely makes sense to remove the word repository.